### ISSUE TITLE
Update for v1 of OSC

### ISSUE THREAD

**Issue by GeneralMidi** [ThrashJazzAssassin/REAPER-OSC-panels#1]
Hi,

Any chance you could update the Reaper OSC panels for v1 and above of Open Stage Control? At the moment it converts it itself but I'm not sure it looks exactly the same....

**Comment by ThrashJazzAssassin** [comment]
Hi GM, I have been working on a V1 updated version customised for my own use. I'll create a general use version when I have time. Thanks for the interest.

**Comment by GeneralMidi** [comment]
Hey thanks for responding. Might I be so bold to ask whether I could take a look at your customised version as is? I'm really loving reaper and trying to learn as much as I can (recent convert from cubase!).

**Comment by ThrashJazzAssassin** [comment]
Yeah sure, here you go. 

[REAPER OpenStageControl TJA 0.2.3looper6.json.zip](https://github.com/ThrashJazzAssassin/REAPER-OSC-panels/files/6320654/REAPER.OpenStageControl.TJA.0.2.3looper6.json.zip)

**Comment by GeneralMidi** [comment]
Legend!

**Comment by GeneralMidi** [comment]
Thanks for sharing your template. It's very good! I think I prefer the original OSC panels template though. Hope you'll get the chance to update it soon. Cheers!!

**Comment by GeneralMidi** [comment]
Quick question. Is it possible to select a track in Reaper, say track 20 and have O-S-C automatically jump to the track?

**Comment by ThrashJazzAssassin** [comment]
Yes, that's possible. Look in Default.ReaperOSC for DEVICE_TRACK_FOLLOWS and DEVICE_TRACK_BANK_FOLLOWS

**Comment by GeneralMidi** [comment]
Hi again, I really love the reaper panels but I need it working for v1 of osc. I've tried the automatic conversion but it looks terrible. I'm happy to help if you don't have so much time to look at it at the moment. It looks like the css variable ``--color-track`` doesn't exist anymore so I'm unsure how to get the beautiful glass look you have. If you point me in the right direction, I'm happy to help. Or maybe you could quickly take a look, it might not take you very long? Cheers.

**Comment by ThrashJazzAssassin** [comment]
[OSC.zip](https://github.com/ThrashJazzAssassin/REAPER-OSC-panels/files/10095320/OSC.zip)
Attached is my latest Reaper panel made with O-S-C 1.20.0

**Comment by ThrashJazzAssassin** [comment]
![osc](https://user-images.githubusercontent.com/15689085/204063605-8d10a5bc-82a1-41e9-ba8d-f69cca35c43d.gif)
